data_io.py

```diff
@@ -0,0 +1,70 @@
+import mimetypes
+import os
+from json import dump
+
+import pandas as pd
+
+
+# Input
+def take_input_data(path: str) -> pd.DataFrame:
+    if (
+        mimetypes.guess_type(path)[0] == "text/plain"
+    ):  # If it's a single file with text type use this block to simply load a df
+        df_data = pd.read_table(
+            path,
+            on_bad_lines="skip",
+            sep=" ",
+            skipinitialspace=True,
+            names=[
+                "Timestamp in seconds since the epoch",
```

Comment:
please move it outside function, this could be turned into "config" variable - list of strings with column names. Place it at least at the beginning of file as PD_COLUMN_NAMES.

Reply:
Good idea. 👍🏻 List would be formatted as this one is by Black though. I just moved it at the beginning. Clearly improved readability.